pkg/cfn/outputs/irsa.go

```diff
@@ -0,0 +1,28 @@
+package outputs
+
+import (
+	cfn "github.com/aws/aws-sdk-go/service/cloudformation"
+)
+
+// ServiceAccount/Role
+type IRSA struct {
+	IAMRole        string
+	ServiceAccount string
+}
+
+var (
+	IRSAOutput chan IRSA
+)
+
+func init() {
```

Comment:
Please use an init method of some kind. We are trying to avoid using `init`. :)

Reply:
Yes, @Skarlso will look at that. Thanks